bluemix/configuration/core_config/bx_config.go

```diff
@@ -676,12 +696,15 @@ func (c *bxConfig) UnsetAPI() {
 		c.data.APIEndpoint = ""
```

Comment:
I think we want to also reset `c.data.IsAccessFromVPC` in the `ClearSession` method, so that flag resets when a user logs out.

Reply:
Tested this out and per discussion, the flag is tied to the API and not a login session. Also confirmed that if the user first sets the API with vpc, but then logs in with `-a private.cloud.ibm.com` and not the vpc setting, it will set the config correctly to remove vpc. Thanks for thinking of this scenario, but looks like everything is correct without a change.